lightly/api/api_workflow_compute_worker.py

```diff
@@ -805,3 +819,82 @@ def selection_config_from_dict(cfg: Dict[str, Any]) -> SelectionConfig:
         strategies.append(SelectionConfigEntry(**entry))
     new_cfg["strategies"] = strategies
     return SelectionConfig(**new_cfg)
+
+
+def worker_config_from_dict(
+    deserialize: Callable,
+    worker_config_dict: Optional[Dict[str, Any]]
+) -> Optional[DockerWorkerConfigV2Docker]:
+    """Converts worker config to DockerWorkerConfigV2Docker instance.
+
+    Args:
+        deserialize:
+            Function to deserialize the worker_config_dict.
+        worker_config_dict:
+            Configuration dict with snake case keys or None.
+
+    Returns:
+        An instance of DockerWorkerConfigV2Docker or None if the input is None.
+        
+    """
+    if worker_config_dict is None:
+        return worker_config_dict
+
+    worker_config_camel_case = _config_to_camel_case(cfg=worker_config_dict)
+    return deserialize(worker_config_camel_case, DockerWorkerConfigV2Docker)
+
+
+def lightly_config_from_dict(
```

Comment:
Can these two function be merged into one? Furthermore, should they be internal and start with `_` ?